tests/inputs/sub-class-name/error_filter_set.py

```diff
@@ -0,0 +1,15 @@
+from nautobot.apps.filters import NautobotFilterSet
+from nautobot.core.models.generics import PrimaryModel
+
+
+class AddressObject(PrimaryModel):
+    pass
+
+
+class MyAddressObjectFilterSet(NautobotFilterSet):
```

Comment:
Is this what the checker should report? @whitej6

Reply:
Yes this should fail